src/CoreFoundation.zig

```diff
@@ -28,6 +170,15 @@ pub const CFData = opaque {
     extern "c" fn CFDataGetLength(*CFData) i32;
 };
 
+/// Wraps CFDateRef type.
+pub const CFDate = opaque {
+    extern "C" fn CFDateCompare(the_date: *CFDate, other_date: *CFDate, context: ?*anyopaque) CFComparisonResult;
+    extern "C" fn CFDateCreate(allocator: ?*CFAllocator, at: CFAbsoluteTime) ?*CFDate;
```

Comment:
How about creating a wrapper for this in Zig assuming a default macOS allocator? Later we could also expose a version of this function which accepts a Zig `Allocator` too.

Reply:
> How about creating a wrapper for this in Zig assuming a default macOS allocator? 

Sure thing - can do that.

> Later we could also expose a version of this function which accepts a Zig Allocator too.

This is potentially trickier, because constructing a `CFAllocator` is itself a non-trivial thing, see my comment recommending:

>  that you construct a CFAllocator from a zig allocator once up-front & re-use it